react-client/dist/style.css

```diff
@@ -1,20 +1,51 @@
+.allThefont {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.topContainer {
```

Comment:
I think the standard is to use selectors like this `.css-selector-name` and you have a mix of both camel case and hyphens